File: conditinals/notes.py
#jasper holt, conditinals notes .py
name=input("what is your name")
#What puts something inside of the “if” statement?
if name=="jasper":
    print("hi jasper")
else:#<-- this happens if condition is false
    print(f"lello{name}")
#(  )white space/ the TAB

#What do if statements do?
#check a condition and if it is true it will do a thing
if name=="jasper": #<---- this is the condition
    print("hi jasper")#<-- whaat it does if true

#What are boolean statements? 
# the part of your conditinal that is eather true or false
if name=="jasper":#<--- this is a boolean statment
    print("hi jasper")
else:#<-- this happens if condition is false
    print(f"lello{name}")

#What do else statements do?

if name=="jasper":#<--- this is a boolean statment
    print("hi jasper")
else:# if the boolean is false the else stament happens
    print(f"lello{name}")

#5.What kind of statement do you use if you have more than 2 needed outcomes?
# computers read things from top to bottom, check the least liky things first
num=int(input("how many cookies are there"))
if num==0:#<-- if always satarts the conditional
    print("there are none.")
elif num==1:
    print("there is one")
elif num<4:#<-- anyhting inbetween id elif
    print("there is a couple")
else :#<-- els always ends the conditinal
    print("there are many")

#6.What do each of the different symbols mean in conditionals?

#< less than
#> greater than
#<= less than or eqal to
#>= greateerthan or equal to
#== equal to
#===*doesent exist in python
#! not
#7.What are the 3 logical operators?
if num<10 and num >5:#ND BOTH BOOLEANS MUST BE TRUE
    print('this is a big single diget number')
elif num<10 or num >5:# or means one must be true
    print('this is a big single diget number')
elif not num<10:#checking to se if statment is false
    print('this is not a big single diget number')
#8.What are logical operators for?
#making our statments more complex

#What does a nested conditional statement do?
if num<10:
    if num==8:
        print("this prints at lest 8")
    else:
    print("the numberis less than 8")
#How do you write an if statement in C?

#How do you write else statements in C?

#How do you write elif/ else if statements in C?

#How do you write the 3 logical operators in C?\

File: conditinals/notes.c
// Jasper Holt, condidtinals notes
#include <stdio.h>
#include <string.h> // needed to compair strings
char name[]="jasper";
int num;

int main(void){
    //10.How do you write an if statement in C?
    if(strcmp(name,"jasper")==0){// betean the curly bracets its in our if stament//strcmpmeans string comparison whne the outcome is 0 the strings are the same
        printf("hello jasper");
    
    //11.How do you write else statements in C?
    }else{
        printf("hello human welcome to my mind\n");
    }
    printf("how many siblings do you have?\n");
    scanf("%d", &num);
    //12.How do you write elif/ else if statements in C?
    if(num==0){
        printf("you are an only child");
    }else if(num <= 2){
        printf("you have a couple siblings");
    }else if (num<=5){
    printf("you jhave a few siblings");
    }else{
        printf("you have a lot of siblings");
    }

    //13.How do you write the 3 logical operators in C?
    // &&= and
    // || = or
    // ! = not

    if(num==7 || num==13){
        printf("that is an unlicky number");
    }else if(num<10 && num>5){
        if(num==12){
            printf("thats a douzen");
        }
        printf("%d is a lagr single diget number\n", num);
    }else{
        printf("you typed in %d\n",num);
    }
    return 0;
}